phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexToolVerificationResult.java

```diff
@@ -48,35 +48,43 @@
         long expiredIndexRowCount = 0;
         long missingIndexRowCount = 0;
         long invalidIndexRowCount = 0;
+        long mismatchingNumberOfCellsCount = 0;
```

Comment:
The isVerificationFailed method needs to be updated accordingly too

Reply:
But the invalidIndexRowCount is updated anyways. Not sure I understand why we need it is isVerificationFailed. This counter and invalidIndexRowCount will be set to 1 if there is 1 such index. @kadirozde